BufferValues/EnergyBar.ini

```diff
@@ -0,0 +1,29 @@
+; Original by SilentNightSound & Satan1c
+
+namespace = global\EnergyBar
+
+[Constants]
+global persist $energy = 0
+global $EnergyActive = 0
+global $energyBarsCounter = 0
+
+[Present]
+post $EnergyActive = 0
+post $energyBarsCounter = 0
+
+[ShaderOverrideEnergyBarPS]
+hash = b0506b5f2797358a
+allow_duplicate_hash=overrule
+
+if $EnergyActive && ps-t0 == 35
+	
+	$energyBarsCounter = $energyBarsCounter + 1
+	
+	if $energyBarsCounter == 1 && $\global\Counter\frames % 31 == 0
+		pre store = $energy, ps-cb0, 33
```

Comment:
I am referring to this file to try to modify the HSR, but I don't understand what the "ps-cb0,33" at line:23 of code means. Where did I find this value?

Reply:
some parts of conversation, with some notes, and screenshots:
https://discord.com/channels/971945032552697897/995556765179596890/1133750298616275067
https://discord.com/channels/971945032552697897/995556765179596890/1133779404615581777
https://discord.com/channels/971945032552697897/995556765179596890/1133793375271784478

ps-cb0, is PS shader memory, with sates values, it placed in line order:
0: 0 1 2 3
1: 4 5 6 7
 . . .
8: 32 33 34 35